docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions.md

```diff
@@ -371,26 +325,26 @@ The default container runtime for OVAs is [Podman](https://podman.io/), not Dock
         </TabItem>
         </Tabs>
 
-21. The output of the script contains credentials and values you will need when completing the installation with the
+18. The output of the script contains credentials and values you will need when completing the installation with the
     Palette CLI. If you need to review this information in the future, invoke the script again.
 
-22. If you have used a release-specific installation OVA, skip this step. Otherwise, if you have used a generic
-    installation OVA, use the following command to execute the Palette VerteX airgap installation binary.
+19. Next, download the third party binary. Your support representative will provide you with credentials to access the
+    third-party binary. Use the following command to download the third-party binary. Replace the `XXXXX` and `YYYYY`
+    placeholders with the credentials provided to you. Replace the `X.X` placeholder with the version of the third-party
+    binary you are downloading. Ask your support representative for the version of the third-party binary you need.
 
     ```shell
-    chmod +x airgap-vertex-<version>.bin && ./airgap-vertex-<version>.bin
+    curl --user XXXXX:YYYYY https://software-private.spectrocloud.com/airgap/thirdparty/airgap-thirdparty-X.X.X.bin \
+    --output airgap-upload.bin
     ```
 
-    Consider the following example for reference.
+20. Use the following command to start the third-party binary. The third-party binary uploads the release-specific packs
+    and images to the registry configured in step **17** of this guide. This process may take some time to complete.
```

Comment:
I may be misinterpreting this, but does the user configure the registry in step 17?

I do see the registries displayed in the example output, but I can't seem to find the step when the registry is configured.

Reply:
@benradstone the script is what initializes and configured the Harbor registry.